lib/rules/max-lines-per-function.js

```diff
@@ -79,13 +79,16 @@ module.exports = {
             OPTIONS_OR_INTEGER_SCHEMA
         ],
         messages: {
-            exceed: "{{name}} has too many lines ({{lineCount}}). Maximum allowed is {{maxLines}}."
+            exceed: "{{name}} has exceeded the limit of lines allowed by ({{linesExceed}}). Maximum allowed is {{maxLines}}."
         }
     },
 
     create(context) {
         const sourceCode = context.getSourceCode();
-        const lines = sourceCode.lines;
+        const lines = sourceCode.lines.map((text, i) => ({
+            lineNumber: i + 1,
+            text
+        }));
```

Comment:
Is this array necessary? It doesn't seem that it provides much value for the rest of the code, so can we just use `sourceCode.lines`?